pac4j-saml/src/main/java/org/pac4j/saml/client/SAML2Client.java

```diff
@@ -212,9 +225,14 @@ protected ChainingMetadataResolver initChainingMetadataResolver(final MetadataRe
         }
         return metadataManager;
     }
+
     @Override
     protected BaseClient<SAML2Credentials, SAML2Profile> newClient() {
-        return new SAML2Client(this.configuration.clone());
+        try {
+			return new SAML2Client(this.configuration.clone());
```

Comment:
I see the whole newClient() method has gone so it's not relevant anymore. However, cloning clone() was not from me.